src/rational/q/from.rs

```diff
@@ -110,6 +110,100 @@ impl FromStr for Q {
     }
 }
 
+impl Q {
+    /// Create a [`Q`] from two references that can be converted to [`Z`].
+    /// For example, [`&Z`].
+    ///
+    /// Warning: The interface of this function will likely change in the future
+    /// or it will entirely be removed. Therefore use [`Q::try_from`] instead.
+    ///
+    /// Parameters:
+    /// - `numerator` of the new [`Q`].
+    /// - `denominator` of the new [`Q`].
+    ///
+    /// Returns a [`Q`] or a [`MathError`]
+    ///
+    /// # Example
+    /// ```ignore (private function)
+    /// use math::rational::Q;
+    /// use math::integer::Z;
+    ///
+    /// let num = Z::from(100);
+    ///
+    /// let a = Q::try_from_int_int_borrow(&num, &i64::MAX).unwrap();
+    /// let b = Q::try_from_int_int_borrow(&num, &i64::MAX).unwrap();
+    /// ```
+    ///
+    /// # Errors and Failures
+    /// - Returns a [`MathError`] of type [`DivisionByZeroError`](MathError::DivisionByZeroError)
+    /// if the denominator is zero.
+    fn try_from_int_int_borrow(
```

Comment:
```suggestion
    fn try_from_int_int(
```
As you do not implement this function for owned types, just leave the `_borrow` at the end out.